server/src/bots/telegram/utils/keyboard.ts

```diff
@@ -0,0 +1,25 @@
+import {getChatId} from "./chat";
+import {Cache} from "../../../utils/cache";
+import {UserMessages} from "../../../models/constants";
+
+const KeyboardWrapper = require("node-telegram-keyboard-wrapper");
+
+export const getKeyboard = (message) => {
```

Comment:
types

Reply:
return some kind of structure
if possible you can type it, otherwise use `export const getKeyboard = (message): unknown => {`
https://github.com/alexandercerutti/node-telegram-keyboard-wrapper#open

Reply:
It accepts and returns **any** type
Does it make sense writing it to the code?